float32[] action
int32 step_n
---
bool success
string message
float32[] next_state
float32 reward
bool done

